vividus-tests/src/main/resources/properties/suite/integration/suite.properties

```diff
@@ -33,3 +33,12 @@ ui.visual.ignored-elements=By.id(some_fake_id)
 # and processed by configuration resolver and decryptor
 property=#{2 * 2}
 bdd.variables.global.property-with-int-value=#{2 + 2}
+
+db.connection.testh2db.driver-class-name=org.h2.Driver
```

Comment:
I do like the idea to have a real-world example of it, but maybe you could reuse the already existing database to not overload CI runner? https://github.com/vividus-framework/vividus/blob/6d5b3a5093539d8aef3fca903ead0bfae237ce24/vividus-tests/src/main/resources/story/system/DatabaseStepsTests.story#L7-L26 
 cc @valfirst

Reply:
I'd like to keep in-memory db:
1) less dependencies on 3-rd party systems
2) example of how to use in-memory DB
3) it's relatively fast and doesn't bring much performance overhead

Reply:
So left as is.